src/coreclr/src/gc/gcinterface.h

```diff
@@ -734,6 +734,9 @@ class IGCHeap {
     // Tells the GC how many YieldProcessor calls are equal to one scaled yield processor call.
     virtual void SetYieldProcessorScalingFactor(float yieldProcessorScalingFactor) = 0;
 
+    // Flush the log and close the file if GCLog is turned on.
+    virtual void ShutDownLogging() = 0;
```

Comment:
```suggestion
    virtual void Shutdown() = 0;
```

I believe that "Shutdown" is a single word. Also, I do not think we need "logging" in name. The GC can do whatever it wants in this method.

Reply:
Thanks for the review.

There are a couple of things that worth notice that I didn't mention earlier.

`GCHeap` does have an existing method named `Shutdown` [here](https://github.com/dotnet/runtime/blob/4f9ae42d861fcb4be2fcd5d3d55d5f227d30e723/src/coreclr/src/gc/gcimpl.h#L73) so the name is claimed. Yet, as far as I can tell, the function is never called. I attempted to call that function instead in the call site I added, that is leading to an AV, I haven't debugged what is happening there yet, not sure if that is something we wanted to fix.

I also tried to name the method `ShutdownLogging` instead of `ShutDownLogging`. I am getting a weird compiler error on Windows complaining there is a syntax error at the ` = ` sign on that line. I have no idea what happened there and renaming the method solved the problem.

I assume the comment is all about making the naming less general and therefore we can extend the shutdown logic without touching the interface, I can try and see what I can do there.

Reply:
> I also tried to name the method ShutdownLogging instead of ShutDownLogging. I am getting a weird compiler error on Windows

That's because of there is `#define ShutdownLogging` in log.h...

Reply:
Ok, you can keep the name as is then.

Reply:
I see a Shutdown method on IGCHandleManager, not on IGCHeap...am I reading this wrong? there is a Shutdown method on the internal GCHeap which is not called. we could get rid of the code in this method's implementation or if we think some day we might be required to actually clean up the resource we could keep the code but not execute it; and only execute the shutdown logging part.